TransifexNativeSDK/clitool/src/test/java/com/transifex/clitool/MainClassTest.java

```diff
@@ -164,6 +165,11 @@ public void testPush_useFile() {
         assertThat(parsedPostData.data.keySet()).containsExactly("tx_test_key", "tx_plural_test_key").inOrder();
         assertThat(parsedPostData.data.get("tx_test_key").string).isEqualTo("test");
         assertThat(parsedPostData.data.get("tx_plural_test_key").string).isEqualTo("{cnt, plural, one {car} two {car 2} other {cars}}");
+
+        // Check appended tag
+        assertThat(parsedPostData.data.get("tx_test_key").meta.tags).isNotNull();
+        assertThat(parsedPostData.data.get("tx_test_key").meta.tags).asList().containsExactly("some_tag");
+        assertThat(parsedPostData.data.get("tx_plural_test_key").meta.tags).asList().containsExactly("some_tag");
```

Comment:
Let's test with multiple tags, not just one at the time.

Reply:
Done!